src/App.js

```diff
@@ -1,20 +1,64 @@
+import { useEffect, useState } from 'react'
+import { v4 as uuidv4 } from 'uuid'
 import styled from 'styled-components'
+
+import FloatingActionButton from './components/FloatingActionButton'
+
+import loadLocally from './lib/loadLocally'
+import AddBookForm from './components/AddBookForm'
+import getBooks from './services/getBooks'
+
 import BookList from './components/BookList'
+import BookCard from './components/BookCard'
 
 export default function App() {
+  const [books, setBooks] = useState(loadLocally('books') ?? [])
+
+  const [addBookFormModal, setaddBookFormModal] = useState(false)
```

Comment:
A small typo: `setAddBookFormModal`
```suggestion
  const [addBookFormModal, setAddBookFormModal] = useState(false)
```